mrcnn/model.py

```diff
@@ -2195,7 +2574,7 @@ def compile(self, learning_rate, momentum):
             layer = self.keras_model.get_layer(name)
             self.keras_model.metrics_names.append(name)
             loss = (
-                tf.reduce_mean(layer.output, keepdims=True)
+                tf.reduce_mean(layer.output, keep_dims=True)
```

Comment:
This change is unnecessary. keep_dims is deprecated.

Reply:
Same as above